docs/using-gitbase/supported-clients.md

```diff
@@ -0,0 +1,3 @@
+## Supported clients
+
+To see the supported MySQL clients and examples about how tho use them, take a look [here](https://github.com/src-d/go-mysql-server/blob/master/SUPPORTED_CLIENTS.md).
```

Comment:
Instead of master, it should point to the tree of the current revision, like we don on some other parts of the doc.

Reply:
I've changed to point to `https://github.com/src-d/go-mysql-server/blob/679d33772845593ce5fdf17925f49f2335bc8356/SUPPORTED_CLIENTS.md`which right now isn't a valid url, but when `go-mysql-server` dependency will be updated it will a point to the correct url.